gensim/models/lsimodel.py

```diff
@@ -348,6 +348,15 @@ def add_documents(self, corpus, chunksize=None, decay=None):
         """
         logger.info("updating model with new documents")
 
+        if isinstance(corpus, list):
+            logger.warning(
+                "taking plain list as a corpus for LSI;"
```

Comment:
Please make a few fixes:
- Use capital letters/dots as a start/end of the sentence.
- Be careful that your words do not stick together, for example
```python
print(
"hello"
"world"
)
```
output is `helloworld`